spec/lib/embed/viewer/image_spec.rb

```diff
@@ -13,9 +13,10 @@
     end
   end
   describe 'self.supported_types' do
-    it 'should return an array of supported types' do
-      expect(Embed::Viewer::Image.supported_types).to eq [:image, :manuscript, :map, :book]
-    end
+    pending(' is pending due to the migration to collection viewer')
```

Comment:
Could we make the pending specs like this?

```ruby
describe 'self.supported_types' do
  pending 'should return an array of supported types' do
    expect(Embed::Viewer::Image.supported_types).to eq [:image, :manuscript, :map, :book    
  end
end

# or

describe 'self.supported_types' do
  it 'should return an array of supported types' do
    pending(' is pending due to the migration to collection viewer')
      expect(Embed::Viewer::Image.supported_types).to eq [:image, :manuscript, :map, :book]
   end
end
```

That way only 1 line is changing?